dockers/docker-orchagent/docker-init.j2

```diff
@@ -34,9 +34,12 @@ if [ -x /usr/share/sonic/hwsku/hwsku-init ]; then
     /usr/share/sonic/hwsku/hwsku-init
 fi
 
-# Start arp_update and NDP proxy daemon when VLAN exists
-if [ "$VLAN" != "" ]; then
+# Start arp update when VLAN exists or switch type as chassis packet for backend port channel interfaces
+if [[ "$VLAN" != "" ]] || [[ "$SWITCH_TYPE" == "chassis-packet" ]]; then
```

Comment:
may be we should have a separate check for "chassis-packet and copy the arp.conf. Let Vlan check be separate.